mne_icalabel/iclabel/tests/test_network.py

```diff
@@ -126,14 +128,58 @@ def test_network_outputs():
     assert np.allclose(matlab_labels, torch_labels, atol=1e-7)
 
 
+def test_network_outputs_onnx():
+    """
+    Compare that the ICLabel network in onnx and matlab outputs the same
+    values for a common set of features (input to the forward pass).
+
+    Notes
+    -----
+    The forward pass has been run in matconvnet with the same input features.
+    The corresponding MATLAB code can be found in 'data/network_output.txt'.
+    """
+    # load features to use for the forward pass
+    features = loadmat(matconvnet_fw_input_path)["input"][0, :]
+    # features is a (6, ) array with:
+    # - elements in position [0, 2, 4] -> 1-element array with the var name
+    # - elements in position [1, 3, 5] -> data arrays
+    assert "in_image" == features[0][0]
+    images = features[1]
+    assert "in_psdmed" == features[2][0]
+    psd = features[3]
+    assert "in_autocorr" == features[4][0]
+    autocorr = features[5]
+
+    # reshape the features to fit torch format
+    images = np.transpose(images, (3, 2, 0, 1)).astype(np.float32)
+    psd = np.transpose(psd, (3, 2, 0, 1)).astype(np.float32)
+    autocorr = np.transpose(autocorr, (3, 2, 0, 1)).astype(np.float32)
+
+    # run the forward pass on onnxruntime
+    ort_session = ort.InferenceSession(onnx_network_file)
+
+    labels = ort_session.run(
+        None,
+        {"topo": images, "psds": psd, "autocorr": autocorr},
+    )
+
+    onnx_labels = labels[0]
+
+    # load the matconvnet output of the forward pass on those 3 feature arrays
+    matlab_labels = loadmat(matconvnet_fw_output_path)["labels"]  # (30, 7)
+
+    # Compare both outputs
+    assert np.allclose(matlab_labels, onnx_labels, atol=1e-7)
+
+
 @pytest.mark.parametrize(
     "eeglab_feature_file, eeglab_feature_formatted_file",
     [
         (features_raw_path, features_formatted_raw_path),
         (features_epo_path, features_formatted_epo_path),
     ],
 )
-def test_format_input(eeglab_feature_file, eeglab_feature_formatted_file):
+def test_format_input_pytorch(eeglab_feature_file, eeglab_feature_formatted_file):
```

Comment:
```suggestion
def test_format_input(eeglab_feature_file, eeglab_feature_formatted_file):
```

This is testing a function common to both. I would keep the old name.